content/en/continuous_integration/setup_pipelines/buildkite.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Set up Tracing on a Buildkite Pipeline
+kind: documentation
+further_reading:
+    - link: "/continuous_integration/explore_pipelines"
+      tag: "Documentation"
+      text: "Explore Pipeline Execution Results and Performance"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+{{< site-region region="us,eu" >}}
+## Configuring the Datadog integration
+
+The Datadog integration for [Buildkite][2] works by using [webhooks][1] to send pipeline data to Datadog.
+
+Go to **Settings > Notification Services** in Buildkite and add a new webhook:
+
+{{< site-region region="us" >}}
+* **Webhook URL**: `https://webhooks-http-intake.logs.datadoghq.com/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
+* **Events**: Select `job.finished` and `build.finished`.
+* **Pipelines**: Select all pipelines or the subset of pipelines you want to trace.
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{< /site-region >}}
+
+{{< site-region region="eu" >}}
+* **Webhook URL**: `https://webhooks-http-intake.logs.datadoghq.eu/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
+* **Events**: Select `job.finished` and `build.finished`.
+* **Pipelines**: Select all pipelines or the subset of pipelines you want to trace.
+
+[1]: https://app.datadoghq.eu/account/settings#api
+{{< /site-region >}}
+
+To set custom `env` or `service` parameters, add more query parameters in the webhooks URL: `&env=<YOUR_ENV>&service=<YOUR_SERVICE_NAME>`.
```

Comment:
As a user, I wouldn't know what `env` and `service` mean in this context

Reply:
I'll add a link to https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes#overview which has the explanation of this reserved env vars.

Reply:
I think we are using the `service` tag as the "CI provider instance id".

Reply:
Do we have that documented somewhere? It seems like it doesn't match the `service` in Data Dog as a whole which I assume would cause confusion for customer using multiple products.

Reply:
I saw that mentioned here: https://docs-staging.datadoghq.com/carlos.gonzalez/add-buildkite-integration-docs/continuous_integration/setup_pipelines/gitlab/?tab=gitlabcom#configuring-the-datadog-integration

Reply:
If buildkite doesn't support an on-prem version, it might just make sense to not document the `service` tag for buildkite.

Reply:
Is interesting because in that case does `env` have any value? We use it to separate between staging and prod and so on, but Buildkite only has one instance so both staging and prod would run from the same Buildkite thus `env` is always the same.

I guess you could setup multiple webhooks for different pipelines with different branch rules but that seems like a lot of work for the customer. You could do the same with services if you wanted to map some pipeline to some service namespace.

I find it a bit weird that our service does not match the general consensus of what service is in data dog.

Reply:
These are for advanced use cases and we always default to empty. If there is no way to configure different instances of Buildkite I'd say they are not needed. 

The plan is to rename `service` to `ci.instance` this is why we have this facet already, but we never go to changing it in query params and docs.

Reply:
Let's remove it from here. And also, we should remove them from the instructions of GitLab.com

Reply:
why would we remove them from Gitlab?

Reply:
From the SaaS section I meant